integration_tests/smoke/prisonNumberGenerator.ts

```diff
@@ -0,0 +1,11 @@
+export default function generateRandomPrisonNumber(): string {
+  const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+  const randomLetter = (): string => {
+    return alphabet[Math.floor(Math.random() * alphabet.length)]
+  }
+  const randomNumber = (): string => {
+    return Math.floor(Math.random() * 10).toString()
+  }
+
+  return `${randomLetter()}${randomNumber()}${randomNumber()}${randomNumber()}${randomNumber()}${randomLetter()}${randomLetter()}`
```

Comment:
There's probably a nicer way to do this!

Reply:
Presumably its to generate a prison number that is (probably) not in the database as a contact? So that the tests can consistently drive the correct/expected screens?
Hmmm, not sure of a better way to do this tbh, other than using a fixed/known prison number, and having some kind of pre-test step that deletes it somehow .... (not that we have an API endpoint for that .... 🤔 )